### PR TITLE
refactor: migrate log-analysis to union es storage

### PR DESCRIPTION
**Pull request by snakorse**
#### What type of this PR

Add one of the following kinds:
/kind refactor

#### What this PR does / why we need it:
1. migrate log-analysis to union es storage
2. expose front-api from msp

#### Specified Reviewers:

/assign @liuhaoyang @recallsong 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      migrate log-analysis to union es storage        |
| 🇨🇳 中文    |        重构日志分析，与容器日志共用一份存储      |

### CODE REVIEW

File: conf/msp/msp.yaml [hunk 1]
```diff
@@ -24,6 +24,17 @@ elasticsearch:
   username: "${ES_SECURITY_USERNAME}"
   password: "${ES_SECURITY_PASSWORD}"
 
+elasticsearch@logs:
```
Comment (liuhaoyang) [inline conf/msp/msp.yaml:27]: msp-service should not directly connect to es to query logs. It is better to query logs through the grpc API of monitor-service

File: modules/msp/apm/log-service/query/log.service.histogram.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+	"github.com/ahmetb/go-linq/v3"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	monitorpb "github.com/erda-project/erda-proto-go/core/monitor/log/query/pb"
+	"github.com/erda-project/erda-proto-go/msp/apm/log-service/pb"
+	"github.com/erda-project/erda/modules/extensions/loghub/index/query"
+	"github.com/erda-project/erda/pkg/common/apis"
+)
+
+func (s *logService) HistogramAggregationFromMonitor(ctx context.Context, req *pb.HistogramAggregationRequest) (*pb.HistogramAggregationResponse, error) {
```
Comment (liuhaoyang) [inline modules/msp/apm/log-service/query/log.service.histogram.go:31]: How are FromMonitor and FromLoghub different?

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: modules/msp/apm/log-service/query/log.service.histogram.go (hunk 2 above)
Comment (snakorse) [reply modules/msp/apm/log-service/query/log.service.histogram.go:31]: FromMonitor query from the union es storage, Fromloghub query from the old log-analysis es storage.

File: conf/msp/msp.yaml (hunk 1 above)
Comment (snakorse) [reply conf/msp/msp.yaml:27]: it's temporary code to compitible with the old log-analysis query logic which not suit to integrate with the logic in monitor, as we turn on the new union es storage, this logic will be removed.

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**